APIGymnastics/items.json

```diff
@@ -1,1685 +0,0 @@
-[
```

Comment:
I moved it after realizing that I had stored it in the incorrect node-modules folder.  Sorry for any confusion this may have caused.